### PR TITLE
add EventHandler, Events, IEventHandler, Individual Events,

### PR DESCRIPTION
**Pull request by Ungiebungie**

### CODE REVIEW

File: AdventureTime/AdventureTime/player.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@ namespace AdventureTime
 {
     public class Player
     {
+        public string Name { get; set; }
```
Comment (stu-millner) [inline AdventureTime/AdventureTime/player.cs:9]: this should be part of a different branch

File: AdventureTime/AdventureTime/Events.cs [hunk 2]
```diff
@@ -4,8 +4,8 @@
 
 namespace AdventureTime
 {
-    interface Events
```
Comment (scottcrawfordhtx) [inline AdventureTime/AdventureTime/Events.cs:7]: Rename to IEvents

File: AdventureTime/AdventureTime/Program.cs [hunk 3]
```diff
@@ -17,14 +17,16 @@ static void Main(string[] args)
 
         private static void Intro()
         {
-            GameBoard gameboard = new GameBoard();
-            gameboard.PrintBoard();
+            
 
-            Console.WriteLine("It's so dark in here, and my cock smells of twink-stink. I've gotta get outta here.");
+            Console.WriteLine("It's so dark in here, and my cock smells of twink-stink. I've gotta get outta here. Oh by the way, my name is:");
             Console.ReadKey();
```
Comment (scottcrawfordhtx) [inline AdventureTime/AdventureTime/Program.cs:23]: Use ReadLine. Also this doesn't give the player a name.

**Review by scottcrawfordhtx — COMMENTED** [review]
See comments

**Review by scottcrawfordhtx — APPROVED** [review]

**Pull request merged**